yarn.lock

```diff
@@ -126,6 +126,11 @@ ajv@^6.10.0, ajv@^6.10.2:
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
 
+ansi-escapes@^3.2.0:
```

Comment:
Why did yarn.lock change? Does the new package have a different set of dependencies than the old one?

Reply:
The old dependencies weren't tracked by yarn before... the package.lock was in `packages/cosmic-swingset/setup`.

Reply:
ah, thanks